external/flume/src/main/scala/org/apache/spark/streaming/flume/FlumeUtils.scala

```diff
@@ -147,7 +147,8 @@ object FlumeUtils {
       storageLevel: StorageLevel
     ): ReceiverInputDStream[SparkFlumeEvent] = {
     createPollingStream(ssc, addresses, storageLevel,
-      DEFAULT_POLLING_BATCH_SIZE, DEFAULT_POLLING_PARALLELISM)
+      ssc.conf.getInt("spark.streaming.receiver.maxRate", DEFAULT_POLLING_BATCH_SIZE),
```

Comment:
Not sure this is valid. maxRate is a max number of events per second. This parameter is a max number of records to fetch in a call.